e2e-tests/portal/cypress/integration/journeys/maker/dashboard/gef-deals/dashboard-pagination.spec.js

```diff
@@ -0,0 +1,59 @@
+const { gefDashboard } = require('../../../../pages');
+
+const mockUsers = require('../../../../../fixtures/mockUsers');
+
+const MAKER_LOGIN = mockUsers.find((user) => (user.roles.includes('maker')));
+
+context('Dashboard Deals pagination controls', () => {
+  let deals;
+
+  before(() => {
+    // [dw] at time of writing, the portal was throwing exceptions; this stops cypress caring
+    cy.on('uncaught:exception', (err, runnable) => {
+      console.log(err.stack);
+      return false;
+    });
+
+    cy.deleteGefApplications(MAKER_LOGIN);
+
+    const dummyDeals = new Array(21).fill('').map((_, i) => ({
+      bankInternalRefName: `abc-${i + 1}-def`,
+      bankId: '956',
+    }));
+
+    cy.insertManyGefApplications(dummyDeals, MAKER_LOGIN)
+      .then((insertedDeals) => { deals = insertedDeals; });
+  });
+
+  it('Dashboard Deals displays 20 results, the total number of items, and working First/Previous/Next/Last links.', () => {
+    // login and go to the dashboard
+    cy.login(MAKER_LOGIN);
+    gefDashboard.visit();
+
+    // deals will be shown in update order, so expect them upsidedown..
+    const page1 = deals.slice(1, 21).reverse();
+    const page2 = [deals[0]];
+
+    // test ininital dashboard page
```

Comment:
small typo

Reply:
I think this predates my work - copied over. Happy for this to stay in to prevent lots of re-runs? Will fix with checker ticket

Reply:
Absolutely 👍